sdk/eventhub/azure-eventhub/samples/README.md

```diff
@@ -1,13 +1,3 @@
----
```

Comment:
As I mentioned OOB, let's have an issue in our backlog (july milestone probably?) to readd this in so we don't forget it.

Reply:
Issue created: https://github.com/Azure/azure-sdk-for-python/issues/11129